modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java

```diff
@@ -1028,6 +1028,10 @@ public boolean hasPermission(
 			CommerceAccount commerceAccount, String actionId)
 		throws PortalException {
 
+		if (commerceAccount == null) {
```

Comment:
Ok, but shouldn't we convert it directly to AccountEntry?

Reply:
@luca-pellizzon I have this changed in the subtask PR that will include changes for commerce-service and commerce-api, as this will require changes to the commerceContext.